apis/r/NAMESPACE

```diff
@@ -81,7 +81,6 @@ importFrom(arrow,RecordBatch)
 importFrom(arrow,concat_arrays)
 importFrom(bit64,as.integer64)
 importFrom(bit64,lim.integer64)
-importFrom(data.table,data.table)
```

Comment:
This only removes an import.  Below you define a new one.  It is not yet here -- did you miss a roxygenize run?

Reply:
I marked as draft b/c I'm still working through some rebase issues but I'll regenerate the docs once I'm sure I didn't leave anything behind.